src/pages/docs/platform/architecture/latency.mdx

```diff
@@ -0,0 +1,165 @@
+
+![Message Delivery Latency Measurement](../../../../images/content/diagrams/latency-delivery.png)
+
+## How latency is measured
+
+Ably employs sophisticated measurement methodologies to accurately capture latency performance across the global platform:
+
+### Round-trip latency measurement
+
+Round-trip latency is measured using real browser monitoring through Uptrends Real Browser Monitoring. This approach provides:
+
+1. **Real browser simulation**: Measurements are conducted using actual browser instances, ensuring realistic network conditions
+2. **Global coverage**: Monitoring points are distributed across the globe using a third party service
+3. **Pub/sub simulation**: Real browsers publish messages and measure the complete round-trip time
+4. **Continuous monitoring**: Measurements are conducted continuously to capture performance across different time periods and load conditions
+
+This methodology ensures that round-trip latency measurements reflect the actual experience of end users accessing Ably through web browsers.
+
+### Message delivery latency measurement
+
+Message delivery latency is measured using Ably's internal quality of service (QOS) application. The measurement process includes:
+
+1. **Continuous message publishing**: Messages are sent continuously to a common Ably channel at a rate of 5 messages per second per region
+2. **Payload inclusion**: Each message contains a payload with region information and precise publish timestamp
+3. **Delivery tracking**: Subscribers across different regions receive and timestamp the messages
+4. **Latency calculation**: Delivery latency is calculated by subtracting the publish timestamp from the delivery timestamp
+5. **Statistical sampling**: The 5 messages per second rate ensures statistically relevant sample sizes across all regions
+
+This methodology provides comprehensive coverage of Ably's global infrastructure and ensures that latency measurements reflect real-world conditions across all datacenters and regions.
+
+### Measurement infrastructure
+
+Both measurement systems operate continuously to provide real-time insights into Ably's latency performance:
+
+- **Automated collection**: Measurements are collected automatically without manual intervention
+- **Real-time analysis**: Latency data is processed and analyzed in real-time
+- **Alerting**: Performance thresholds trigger alerts when latency exceeds targets
+- **Historical tracking**: Long-term trends are maintained for performance optimization
+
+### Statistical analysis
+
+Latency data is analyzed across multiple percentiles to ensure comprehensive performance understanding:
+
+- **P50 (median)**: Represents typical performance under normal conditions
+- **P90**: Ensures 90% of messages meet performance targets
+
+This multi-percentile approach ensures that Ably's latency performance is consistent and predictable across all use cases and load conditions.
+
+## Global infrastructure optimization
```

Comment:
From this point forwards, isn't this dovered in our other performance docs? Shoud this be one doc / multiple?